desc/grid.py

```diff
@@ -814,23 +792,12 @@ def ocs(L):
 
         for iring in range(L // 2 + 1, 0, -1):
             ntheta = 2 * M + np.ceil((M / L) * (5 - 4 * iring)).astype(int)
+            if self.sym:
+                ntheta += 1
             dtheta = 2 * np.pi / ntheta
             theta = np.linspace(0, 2 * np.pi, ntheta, endpoint=False)
-            if rotation in {None, False}:
```

Comment:
Do we no longer want explicit sin/cos symmetry for the concentric grids as an option?

Reply:
Daniel decided to remove it here a69b982 since it wasn't used

Reply:
@ddudt I'm fine with removing it, just not sure if we think it might be useful in the future.